executor/executor_test.go

```diff
@@ -2306,3 +2306,13 @@ func (s *testSuite) TestTableScanWithPointRanges(c *C) {
 	tk.MustExec("insert into t values(1), (5), (10)")
 	tk.MustQuery("select * from t where id in(1, 2, 10)").Check(testkit.Rows("1", "10"))
 }
+
+func (s *testSuite) TestUnsignedPk(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
+
+	tk.MustExec("drop table if exists t")
+	tk.MustExec("create table t(id bigint unsigned primary key)")
+	tk.MustExec("insert into t values(9223372036854775810), (9223372036854775812), (1), (2)")
```

Comment:
Why use 9223372036854775810, it's a magic number, any comments?

Reply:
Just a number which bigger than MaxInt64.

Reply:
How about MaxInt64 + 1